lib/temporal/workflow/errors.rb

```diff
@@ -52,6 +52,17 @@ def self.generate_error(failure, default_exception_class = StandardError)
         end
       end
 
+      def self.generate_error_for_child_workflow_start(event)
+        if event.attributes.cause == :START_CHILD_WORKFLOW_EXECUTION_FAILED_CAUSE_WORKFLOW_ALREADY_EXISTS
```

Comment:
`START_CHILD_WORKFLOW_EXECUTION_FAILED_CAUSE_WORKFLOW_ALREADY_EXISTS` is probably a constant in the protobuf, can you please reference it instead of a hardcoded symbol?

Reply:
It's mildly cumbersome - the protobuf constant is an Integer, and you have to call an extra method to resolve it to a symbol. I'll post the change - lmk if you want me to put it back the old way.